examples/new_tasks/add_to_template/f1/general.yaml

```diff
@@ -0,0 +1,24 @@
+group:
+  - f1
+task: f1
+dataset_path: csv
+dataset_name:
+output_type: generate_until
+validation_split: validation
+fewshot_split: null
+test_split: null
+doc_to_text: "{{question}}"
+doc_to_target: "{{answer}}"
+metric_list:
+  - metric: !function utils.f1
+    aggregation: mean
+    higher_is_better: true
+
+repeats: 1
+num_fewshot: 0
+#filter_list:
+#  - name: "get-answer"
+#    filter:
+#      - function: "regex"
+#        regex_pattern: r"\beat"
+#      - function: "take_first"
```

Comment:
How come these are commented out?

Reply:
It's used to filter model output, e.g. take only number in output.
No use in current file. Removed